configs/sandbox_demo/secure.conf

```diff
@@ -0,0 +1,13 @@
+source $BYOBU_PREFIX/share/byobu/profiles/tmux
+
+# setup secure session
+new-session -s secure -n diagnostic -d
+send-keys 'glances' Enter
+
+setenv FOO "foo"
```

Comment:
do we still need this one? :D

Reply:
As discussed we'll leave it so it reflects the original file :-)